components/Slider/src/MDCSlider.m

```diff
@@ -209,6 +213,24 @@ - (UIColor *)backgroundTrackTickColorForState:(UIControlState)state {
   return color;
 }
 
+- (void)setThumbRadius:(CGFloat)thumbRadius forState:(UIControlState)state {
+  _thumbRadiusesForState[@(state)] = [NSNumber numberWithFloat:thumbRadius];
```

Comment:
Should we NSAssert if they assign a negative value?

Reply:
Done

Reply:
Have instead decided to just always use the absolute value.